src/Dialect/ONNX/ShapeInference/TopK.cpp

```diff
@@ -48,9 +48,9 @@ LogicalResult ONNXTopKOpShapeHelper::computeShape(
       outputDims.emplace_back(XBounds.getDim(i));
   }
 
-  // There are two outputs: one for values and one for indices.
```

Comment:
This change makes the patch no longer NFC?
Can we do this change in a different PR?

Reply:
This is required to apply the template for the TopK operator so I prefer to keep it here. I can remove the "NFC".